ray-operator/apis/ray/v1alpha1/rayservice_types.go

```diff
@@ -47,13 +47,13 @@ type ServeConfigSpec struct {
 
 // RayActorOptionSpec defines the desired state of RayActor
 type RayActorOptionSpec struct {
-	RuntimeEnv        map[string][]string `json:"runtimeEnv,omitempty"`
-	NumCpus           *float64            `json:"numCpus,omitempty"`
-	NumGpus           *float64            `json:"numGpus,omitempty"`
-	Memory            *int32              `json:"memory,omitempty"`
-	ObjectStoreMemory *int32              `json:"objectStoreMemory,omitempty"`
-	Resources         map[string]string   `json:"resources,omitempty"`
-	AcceleratorType   string              `json:"acceleratorType,omitempty"`
+	RuntimeEnv        string            `json:"runtimeEnv,omitempty"`
```

Comment:
In this case, we have to embed a json by ourselves? This could be problematic.

Reply:
There are some ways:
1. Use a json
2. Use a config map: either a file or a json

For RuntimeEnv, we still could make it a struct.
But for user_resource field, which is a free json. I think it is better to use string like this.

Also the controller does not use the field like RuntimeEnv itself. It just redirects the config to Ray Dashboard.